<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TRemote</class>
 <widget class="QMainWindow" name="TRemote">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TRemote</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>261</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Heating Power %</string>
    </property>
    <widget class="QLineEdit" name="powerPercentageEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>61</width>
       <height>29</height>
      </rect>
     </property>
     <property name="toolTip">
      <string extracomment="Enter Values between 0.0 and 100.0"/>
     </property>
     <property name="statusTip">
      <string extracomment="Enter Values between 0.0 and 100.0"/>
     </property>
     <property name="whatsThis">
      <string extracomment="Enter Values between 0.0 and 100.0"/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_Set">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>60</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
    <widget class="QPushButton" name="applyButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>91</width>
       <height>29</height>
      </rect>
     </property>
     <property name="toolTip">
      <string extracomment="Apply Changes"/>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="powerPercentageReadEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>61</width>
       <height>29</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_Read">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>60</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Read</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="connectionGroupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>261</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Sever Connection</string>
    </property>
    <widget class="QLabel" name="label_Server">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>60</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Address</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="serverAddressEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>171</width>
       <height>29</height>
      </rect>
     </property>
     <property name="toolTip">
      <string extracomment="Enter Server Address"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="autoSearchButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>91</width>
       <height>29</height>
      </rect>
     </property>
     <property name="toolTip">
      <string extracomment="Start an Automatic Search"/>
     </property>
     <property name="text">
      <string>Auto Search</string>
     </property>
    </widget>
    <widget class="QPushButton" name="manualButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>91</width>
       <height>29</height>
      </rect>
     </property>
     <property name="toolTip">
      <string extracomment="Connect to the Specified Address"/>
     </property>
     <property name="text">
      <string>Manual</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>362</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
